snapshot-labs/sx-monorepo
# fix: don't load starknet votes from hub

### Summary



Extracted from #383 

When connected with a starknet account, avoid loading account's vote on offchain networks, as starknet address ca not vote on offchain.

This will save a http request, which would always return an error

### How to test

1. Connect with a starknet account
2. Go to an offchain space
3. It should not send a graphql votes request to the hub, to load the list of proposals you have voted on this space (for the VOTED ON indicator)
